### DISCUSSION TITLE
Using Gregorian and Islamic Lunar Calendar inside PersianCalendar control and custom month names

### DISCUSSION THREAD

**Discussion by hussain1368** [HandyOrg/HandyControl#851] [Q&A] [answered]
Hello again,
Thanks for replying the previous question. I have a few more:
1. I see here: https://github.com/ghost1372/PersianToolkit there is an option for switching into Gregorian calendar which is a great feature that makes us able to use same control for both calendar types in a multi-culture app, but it does not work for me. I guess the code has changed in the **HandyControls** since **`SetLanguage`** function is no longer available.
`ConfigHelper.Instance.SetLanguage(ConfigHelper.Language.Persian);`
2. Again in the **PersianToolkit github page** I see something mentioned about Lunar calendar. Can we switch the **PerisanCalendar** control to display lunar calendar or it is only about converting dates?
3. Is there any possibility to use custom month names for **PersianCalendar**? I would like to have **"Dari"** month names like: (حمل، ثور، جوزا، سرطان، اسد، سنبله...) ??? May be creating a new control by extending form **PersianCalendar** make it possible?

**Comment by ghost1372** [comment, marked as answer]
Hi,
First of all, I must say that PersianToolkit project is no longer supported and has been archived.
1. I really do not remember whether this feature is currently supported or not because the code has changed 😅, If it does not cause a problem in the program, I will try to add this feature.
2. This is about date conversion and has nothing to do with the calendar.
3. it's a good idea, I will try to find a way to customize it.

Please create a separate issue [here](https://github.com/ghost1372/HandyControls) for the first and third case

**Reply by hussain1368** [reply]
Thanks a lot for your response.
Created the issue as feature request:
https://github.com/ghost1372/HandyControls/issues/70

**Reply by ghost1372** [reply]
thank you
